core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -571,7 +571,8 @@ public LogId getLogId() {
 
   @Override
   public ListenableFuture<ServerStats> getStats() {
-    ServerStats.Builder builder = new ServerStats.Builder()
+    ServerStats.Builder builder
```

Comment:
nit: please consider following the rectangle rule, and wrap lines such that the obey it.  In this case, `.setListenSockets(transportServer.getListenSockets());` should be indented.